src/log.js

```diff
@@ -481,6 +516,52 @@ export class Log {
       error.message = error.message.replace(USER_ERROR_SENTINEL, '');
     }
   }
+
+  /**
+   * @param {!Array} args
+   * @return {!Array}
+   * @private
+   */
+  maybeExpandArguments_(args) {
```

Comment:
I need to re-review this part.

Reply:
Sounds good. Addressed your other comments. PTAL.